### PR TITLE
[Ready] feature/side-nav

### PR DESCRIPTION
**Pull request by detautama**
image

### CODE REVIEW

File: admin/src/app/admin/navigation/navigation.component.html [hunk 1]
```diff
@@ -1,38 +1,49 @@
...
-                                title="Content types"></app-navigation-list-item>
+                                title="{{isExpanded? 'Content types':''}}"></app-navigation-list-item>
       <app-navigation-list-item routerLink="/_/admin/themes"
                                 icon="web"
-                                title="Themes"></app-navigation-list-item>
+                                title="{{isExpanded? 'Themes':''}}"></app-navigation-list-item>
       <app-navigation-list-item routerLink="/_/admin/settings"
                                 icon="settings"
-                                title="Settings"></app-navigation-list-item>
+                                title="{{isExpanded? 'Settings':''}}"></app-navigation-list-item>
```
Comment (DennisAlund) [inline admin/src/app/admin/navigation/navigation.component.html]: Perhaps we can pass the `isExpanded` parameter into the `app-navigation-list-item` instead and let the logic of displaying/hiding the title be in there instead.

**navigation.component.html**
```html
<app-navigation-list-item routerLink="/_/admin/settings"
                          icon="settings"
                          isExpanded="{{isExpanded}}"
                          title="Settings"></app-navigation-list-item>
```

**navigation-list-item.component.html**
```html
<a mat-list-item routerLink="{{routerLink}}" routerLinkActive="active">
  <mat-icon mat-list-icon>{{icon}}</mat-icon>
  <span *ngIf="isExpanded">{{title}}</span>
</a>
```

File: admin/src/app/admin/navigation/navigation-list-item/navigation-list-item.component.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { Component, OnInit, Input } from '@angular/core';
 })
 export class NavigationListItemComponent implements OnInit {
   @Input() icon: string;
+  @Input() isExpanded: string;
```
Comment (DennisAlund) [inline admin/src/app/admin/navigation/navigation-list-item/navigation-list-item.component.ts]: `isExpanded` should be a boolean

File: admin/src/app/admin/navigation/navigation.component.html (hunk 1 above)
Comment (detautama) [reply admin/src/app/admin/navigation/navigation.component.html]: Fixed in commit https://github.com/oddbit/tanam/pull/34/commits/e35287bcb00a28137adb85c4221b7e403f47fdf5

File: admin/src/app/admin/navigation/navigation-list-item/navigation-list-item.component.ts (hunk 2 above)
Comment (detautama) [reply admin/src/app/admin/navigation/navigation-list-item/navigation-list-item.component.ts]: Fixed in commit https://github.com/oddbit/tanam/pull/34/commits/f20f90e97f0fbdfdf949a00cc35167170f61f163

**Pull request merged**